src/modules/previewpane/SvgThumbnailProvider/SvgThumbnailProvider.cs

```diff
@@ -170,24 +178,31 @@ public Bitmap GetThumbnail(string content, uint cx)
                     // WebView2.NavigateToString() limitation
                     // See https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2.navigatetostring?view=webview2-dotnet-1.0.864.35#remarks
                     // While testing the limit, it turned out it is ~1.5MB, so to be on a safe side we go for 1.5m bytes
-                    if (wrappedContent.Length > 1_500_000)
+                    SvgContentsReady.Wait();
+                    if (string.IsNullOrEmpty(SvgContents) || !SvgContents.Contains("svg"))
+                    {
+                        thumbnailDone.Set();
+                        return;
+                    }
+
+                    if (SvgContents.Length > 1_500_00)
```

Comment:
I don't know what those underscores do, but there are 4 zeros here while 5 zeros before. Does that matter?

Reply:
The underscore is used as thousand separator.

Reply:
ooh, nice catch, thank you!